ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -126,9 +132,24 @@ objects:
 
         # test specific code
         # don't let clients impact the global kubeconfig
-        mkdir -p /go/src/github.com/openshift/
-        cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
-        cd /go/src/github.com/openshift/openshift-azure/
+        if [[ "${TEST_IMAGE}" == *"/bushslicer"* ]]; then
+          TEST_NODE_OPTS="etcd:master:node"
+          TEST_CLUSTER="${RESOURCEGROUP}.eastus.cloudapp.azure.com"
```

Comment:
Let's avoid hardcoding eastus here, eventually we should not be setting AZURE_REGION and have the test pick one at random (we already do that in the upgrade test). Can't you read fqdn from containerservice.yaml in `/tmp/shared/openshift-azure`?

Reply:
If you are going to move this into a script in the bushslicer repo, then expect in the script that TEST_CLUSTER is already set and set it here before running TEST_COMMAND.

Reply:
I will fix the region bits.  I'm not sure the extra script will go into the bushslicer repository.  Since this is pretty specific to our use case.  Would placing these variables at the top in the parameters list be acceptable?

Reply:
I attempted to make these variables parameters but ran into an issue with it not being available in the test environment:
In the parameters:
```
- name: BUSHSLICER_DEFAULT_ENVIRONMENT
  value: online_starter
```

Error in test:
```
    Given I have a project                                                    # features/step_definitions/project.rb:7
      please specify default environment key in config or BUSHSLICER_DEFAULT_ENVIRONMENT env variable (RuntimeError)
      /home/jenkins/bushslicer/lib/world.rb:95:in `env'
      /home/jenkins/bushslicer/lib/world.rb:147:in `user'
      /home/jenkins/bushslicer/features/step_definitions/project.rb:21:in `/^I have a project$/'
      features/cli/build.feature:6:in `Given I have a project'
```

Reply:
Once passing the parameter variables to the container through environment variables this worked.
```
      - name: BUSHSLICER_DEFAULT_ENVIRONMENT
        value: ${BUSHSLICER_DEFAULT_ENVIRONMENT}
```